projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts

```diff
@@ -293,4 +293,8 @@ export const IDENTIFIERS_TO_REPLACE: ReplacementIdentifier[] = [
         from: {name: 'TuiFlagPipeModule', moduleSpecifier: '@taiga-ui/core'},
         to: {name: 'TuiFlagPipe', moduleSpecifier: '@taiga-ui/core'},
     },
+    {
+        from: {name: 'TuiRootModule', moduleSpecifier: '@taiga-ui/core'},
```

Comment:
It would break `importProvidersFrom`

Reply:
I added comment to https://github.com/taiga-family/taiga-ui/issues/5808